crits/core/api.py

```diff
@@ -634,7 +634,81 @@ def obj_update(self, bundle, **kwargs):
         :returns: NotImplementedError if the resource doesn't override.
         """
 
-        raise NotImplementedError('You cannot currently update this object through the API.')
+        import crits.actors.handlers as ah
+        import crits.campaigns.handlers as ch
+
+        actions = {
+            'Common': {
+                'campaign_add': ch.campaign_add,
+                'campaign_edit': ch.campaign_edit,
+                'campaign_remove': ch.campaign_remove,
+            },
+            'Actor': {
+                'update_actor_tags': ah.update_actor_tags,
+                'attribute_actor_identifier': ah.attribute_actor_identifier,
+                'set_identifier_confidence': ah.set_identifier_confidence,
+                'remove_attribution': ah.remove_attribution,
+                'set_actor_name': ah.set_actor_name,
+                'set_actor_description': ah.set_actor_description,
+                'update_actor_aliases': ah.update_actor_aliases,
+            },
+            'Campaign': {
+                'add_campaign': ch.add_campaign,
+                'remove_campaign': ch.remove_campaign,
+                'add_ttp': ch.add_ttp,
+                'edit_ttp': ch.edit_ttp,
+                'remove_ttp': ch.remove_ttp,
+                'update_campaign_description': ch.update_campaign_description,
+                'modify_campaign_aliases': ch.modify_campaign_aliases,
+            },
+        }
+
+        path = bundle.request.path[:-1].split('/')
```

Comment:
Is there a programmatic way to get the part of the API being accessed instead of string mangling? Valid (though odd) paths such as "crits/api/v1/emails///" would probably cause issues here.